assets/js/controllers/navigation.controller.js

```diff
@@ -87,27 +87,33 @@ function NavigationCtrl ($scope, $window, $rootScope, $interval, $timeout, $cook
 
   $scope.logout = () => {
     let isSynced = Wallet.isSynchronizedWithServer();
+    let needsBackup = !Wallet.status.didConfirmRecoveryPhrase;
 
-    let options = { friendly: true, cancel: 'NO_THANKS', modalClass: 'top' };
+    let options = (ops) => angular.merge({ friendly: true, modalClass: 'top' }, ops);
     let saidNoThanks = (e) => e === 'cancelled' ? $q.resolve() : $q.reject();
-    let setSurveyCookie = () => $cookies.put('logout-survey', true);
-    let openSurvey = () => { $window.open('https://blockchain.co1.qualtrics.com/SE/?SID=SV_7PupfD2KjBeazC5'); };
+    let hasNotSeen = (id) => !$cookies.get(id);
+    let rememberChoice = (id) => () => $cookies.put(id, true);
 
-    let promptSurvey = () =>
-      Boolean($cookies.get('logout-survey')) === true
-        ? $q.resolve()
-        : Alerts.confirm('SURVEY_CONFIRM', options)
-          .then(openSurvey, saidNoThanks).then(setSurveyCookie);
+    let goToBackup = () => $q.all([$state.go('wallet.common.settings.security'), $q.reject('backing_up')]);
```

Comment:
Wasn't sure which was more appropriate. Am fine with directing to security center instead.